Spanner/src/Transaction.php

```diff
@@ -545,7 +571,8 @@ public function commit(array $options = [])
         }
 
         $options += [
-            'mutations' => []
+            'mutations' => [],
+            'returnCommitStats' => false
```

Comment:
If you make the change below this, you can remove this line.

Reply:
Done.